src/bundle/Resources/config/routing.yml

```diff
@@ -418,6 +418,13 @@ ezplatform.version.remove:
     defaults:
         _controller: 'EzPlatformAdminUiBundle:Version:remove'
 
+ezplatform.version.has_conflict:
+    path: /version/has-conflict/{contentId}/{versionNo}/{languageCode}
```

Comment:
I'm still not convinced with the naming (sorry for bringing this one more time). *Has conflict* with 200 status is kinda misleading as I'd assume 200 is a confirmation. cc @lserwatka

Reply:
Yes, @webhdx is right. 200 would indicate there is a conflict. So to retain this name, we would have to change a logic in the code. Or change a route name.

Reply:
Ok, I changed route and action name to "hasNoConflict"